clarity/src/vm/analysis/type_checker/natives/mod.rs

```diff
@@ -618,8 +618,8 @@ impl TypedNativeFunction {
             CmpGeq | CmpLeq | CmpLess | CmpGreater => {
                 Simple(SimpleNativeFunction(FunctionType::ArithmeticComparison))
             }
-            Sqrti | Log2 => Simple(SimpleNativeFunction(FunctionType::ArithmeticUnary)),
-            Modulo | Power | BitwiseXOR => {
+            Sqrti | Log2 | BitwiseNOT => Simple(SimpleNativeFunction(FunctionType::ArithmeticUnary)),
+            Modulo | Power | BitwiseXOR | BitwiseAND | BitwiseOR | BitwiseLShift | BitwiseRShift | BitwiseXOR2 => {
```

Comment:
Maybe the shift operations should always take a `uint` for the second parameter, regardless of the type of the first?

Reply:
Sounds reasonable! Please have a look at my implementation, I didn't see an existing function type that covered this case, so I created a new one (but I could also be blind) - if there was an easier way to achieve this please let me know :)  Updated type checker tests to account for this.

Reply:
Followup in my comment https://github.com/stacks-network/stacks-blockchain/pull/3382#issuecomment-1308356767 re: no existing function type.